NICE.Identity.Authentication.Sdk/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -36,6 +41,24 @@ public static AuthenticationBuilder AddAuthentication(this IServiceCollection se
 			services.AddScoped<IAuthenticationService, AuthenticationService>();
             services.TryAddScoped<IAPIService, APIService>();
             services.AddHttpContextAccessor();
+            services.AddSingleton<ApiTokenClient>();
+
+            if (authConfiguration.RedisConfiguration != null && authConfiguration.RedisConfiguration.Enabled)
+            {
+                var redisConfiguration =
+                    StackExchange.Redis.ConfigurationOptions.Parse(
+                        authConfiguration.RedisConfiguration.ConnectionString); 
+
+                services.AddSingleton<IRedisCacheClient, RedisCacheClient>();
+                services.AddSingleton<IRedisCacheConnectionPoolManager, RedisCacheConnectionPoolManager>();
+                services.AddSingleton<ISerializer, NewtonsoftSerializer>();
+
+                services.AddSingleton((provider) => provider.GetRequiredService<IRedisCacheClient>().GetDbFromConfiguration());
```

Comment:
isn't this provided by line 52?

Reply:
That line sets up the default database from configuration using the GetDbFromConfiguration method.